src/pages/Technician/index.js

```diff
@@ -1,15 +1,16 @@
 import React from 'react'
-import { Switch, Route } from 'react-router-dom'
+import { Switch, Route, Redirect } from 'react-router-dom'
 
 import AddTechnician from './AddTechnician'
 import EditTechnician from './EditTechnician'
-import Manage from './Manage'
+import Manager from './Manager'
 
 const Technician = () => (
   <Switch>
     <Route exact path="/logged/technician/add" component={AddTechnician} />
-    <Route exact path="/logged/technician/edit" component={EditTechnician} />
-    <Route exact path="/logged/technician/manager" component={Manage} />
+    <Route path="/logged/technician/edit" component={EditTechnician} />
```

Comment:
para usar o id lá na tela de detalhes você precisa passar o id aqui, dessa forma fica disponível no `match.params.id`
```suggestion
    <Route path="/logged/technician/edit/:id" component={EditTechnician} />
```